internal/plugins/golang/init.go

```diff
@@ -43,11 +50,118 @@ func (p *initPlugin) Run() error {
 		return err
 	}
 
+	// Update the scaffolded Makefile with operator-sdk recipes.
+	// TODO: rewrite this when plugins phase 2 is implemented.
+	if err := initUpdateMakefile(); err != nil {
+		return fmt.Errorf("error updating Makefile: %v", err)
+	}
+
 	// Update plugin config section with this plugin's configuration.
 	cfg := Config{}
 	if err := p.config.EncodePluginConfig(pluginConfigKey, cfg); err != nil {
 		return fmt.Errorf("error writing plugin config for %s: %v", pluginConfigKey, err)
 	}
 
+	// Write a kustomization.yaml to the config directory.
+	if err := kustomize.Write(filepath.Join("config", "bundle"), bundleKustomization); err != nil {
+		return err
+	}
+
 	return nil
 }
+
+// kustomization for bundles.
+const bundleKustomization = `resources:
```

Comment:
Could we add a note for we try to improve its structure and have something that will follow up the KB layout?

 E.g `pkg/scaffold/internal/templates/bundle/makefile.go` where would have the new bundle targets. Since we should not change the existent ones just add new we could add at the end the new targets in this scenario. 



However, it might increase the complexity. 



(Out of teh scope here, we can shape as required in the futire)

Reply:
These should live here now, then can be moved if we change anything else about the Makefile.